api/web3server.go

```diff
@@ -435,11 +436,11 @@ func (svr *web3Handler) call(in *gjson.Result) (interface{}, error) {
 }
 
 func (svr *web3Handler) estimateGas(in *gjson.Result) (interface{}, error) {
-	from, to, gasLimit, gasPrice, value, data, err := parseCallObject(in)
+	callMsg, err := parseCallObject(in)
 	if err != nil {
 		return nil, err
 	}
-
+	from, to, gasLimit, gasPrice, value, data := callMsg.From, callMsg.To, callMsg.Gas, callMsg.GasPrice, callMsg.Value, callMsg.Data
```

Comment:
from, to := callMsg.From, callMsg.To
the other 4 can directly use `callMsg.xxx` at L457 below

Reply:
fixed